sdl2promela/translator.py

```diff
@@ -1319,6 +1359,58 @@ def __generate_statement(
     )
 
 
+@dispatch(Context, sdlmodel.Transition, sdlmodel.TransitionChoice)
+def __generate_statement(
+    context: Context,
+    transition: sdlmodel.Transition,
+    transition_choice: sdlmodel.TransitionChoice,
+) -> promelamodel.Statement:
+    builder = SwitchBuilder()
+    state_variable_name = __get_state_variable_name(context)
+
+    for state, transition_id in transition_choice.candidates.items():
+        if isinstance(transition_id, str):
```

Comment:
Why this check? Isn't id an int?

Reply:
I think this at least needs a comment...

Reply:
comment added